prog/dftb+/lib_timedep/linrespgrad.F90

```diff
@@ -583,7 +593,15 @@ subroutine LinRespGrad_old(tSpin, this, iAtomStart, grndEigVecs, grndEigVal, scc
       end if
 
       ! set up transition indexing
-      call rindxov_array(win, mLUMO, nxov, getij, iatrans)
+      call rindxov_array(win, nxov, nxoo, nxvv, getia, getij, getab, iatrans)
+
+   #:block DEBUG_CODE
```

Comment:
Good! (like this)
PR #719 added a unit test framework, so this could also migrate there eventually.

Reply:
ok!